specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.0/keyvault.json

```diff
@@ -298,7 +298,7 @@
             "format": "int32",
             "minimum": 1,
             "maximum": 25,
-            "description": "Maximum number of results to return in a page. If not specified the service will return up to 25 results."
+            "description": "Optional. Specifies the maximum number of results to return in a page. If not specified, then the service uses a default of 25 as an upper bound. Setting maxresults to a value less than 1 or greater than 25 results in error response code 400 (Bad Request). If there are additional results to return, then the service returns a nextLink containing a skip token for pagination. In certain cases, the service might return fewer results than specified by maxresults (even 0 results) and also return a nextLink. Clients should not make any assumptions on the minimum number of results per page, and should enumerate all pages until the nextLink becomes null."
```

Comment:
Separately, don't separate document defaults. It's already documented, we don't want them in the SDK (tight coupling), and the server should return a good error result that says.

Defaults have a tendency to change and this shouldn't be codified in a definition that doesn't typically.

Reply:
Sure, I can remove the part mentioning 25 as the default. But in what location does it get documented separately? When I check the following documentation link, this description field is the only place that mentions the number 25: https://learn.microsoft.com/en-us/rest/api/keyvault/keys/get-keys/get-keys?tabs=HTTP

Reply:
I talked with @johanste offline, and while there is cause for concern about documentation drifting from implementation e.g., the SDK is generated with current defaults, but someone uses the SDK against an older `api-version` where the defaults are different, it's trivial enough that this particular improvement to documentation is probably fine. Consider it, but it's your team's call.

Reply:
Thanks Heath. I agree with your reasoning, and I've removed mentioning the default value from the description.